front-end/src/components/Navigation.js

```diff
@@ -1,12 +1,12 @@
 import React from "react";
-import { AppBar, Toolbar, Link, Box } from "@mui/material";
+import { AppBar, Toolbar, Link, Box, Stack } from "@mui/material";
+import Avatar from "@mui/material/Avatar";
 
 const pages = [
 	{ name: "HOME", link: "/" },
 	{ name: "COURSES", link: "/courses" },
 ];
-
-function Navigation() {
+function Navigation({ isLogin = true }) {
```

Comment:
I am not sure why you did { isLogin = true }. I think you meant Navigation({ isLogin }). ie  can be false or true from Where Navigation component is invoked

Reply:
For isLogin, Delvin suggested I leave it to be true.

Reply:
Additional space will be reviewed and added. Prettier reformed it. 
Thank you